carbonmark-data/components/charts/helpers/DataTable/ClientWrapper/index.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+import Pagination from "components/charts/helpers/DataTable/Pagination";
+import Table from "components/charts/helpers/DataTable/Table";
+import { useState } from "react";
+import { ConfigurationKey } from "../configurations";
+import styles from "./styles.module.scss";
+
+export default function ClientWrapper(props: {
```

Comment:
I don't like the name of this component; it is too abstract and generic IMO

Reply:
done in 03a74f38b